src/config/dynamic-theme-fixes.config

```diff
@@ -3693,6 +3693,22 @@ article, article section, article footer, .css-1fzo8jw, .css-efdc07, article .cs
 
 ================================
 
+brack.ch
+
+CSS
+body, body > header, .popover {
```

Comment:
Setting that breaks some hover color changes. As we don't want main background switched to white on hover then I would change this line to:
`body, body > header:not(:hover), .popover:not(:hover)`

Reply:
Setting this seems to create some additional issues. Hovering over the header part of the website creates a thin white bottom line. Likewise with the popover when logged into an account, it creates a ugly white border breaking the entire cohesion. I am unfortunately bad in CSS, so I am unsure how else this could be fixed.

![popover using :not(:hover)](https://github.com/darkreader/darkreader/assets/119841008/b44bc0c0-a304-4f41-82d4-849ffaf4030e)